df_py/predictoor/csvs.py

```diff
@@ -70,7 +74,73 @@ def loadPredictoorData(csv_dir: str, chainid: int) -> Dict[str, PredictoorBase]:
     return predictoor_data
 
 
+def loadAllPredictoorData(csv_dir: str) -> Dict[str, PredictoorBase]:
+    predictoor_data = {}
+
+    csv_files = glob.glob(os.path.join(csv_dir, "predictoordata-*.csv"))
+    for csv_file in csv_files:
+        # extract chainid from filename
+        match = re.search(r"predictoordata-(\d+)\.csv$", csv_file)
+        if match:
+            chainid = int(match.group(1))
+            predictoor_data.update(loadPredictoorData(csv_dir, chainid))
+
+    return predictoor_data
+
+
 @enforce_types
 def predictoorDataFilename(csv_dir, chainid):
-    f = f"predictoordata_{chainid}.csv"
+    f = f"predictoordata-{chainid}.csv"
+    return os.path.join(csv_dir, f)
+
+
+# ------------------------------- REWARDS -------------------------------
```

Comment:
Having "OCEAN" in the rewards filename comes from a time when we were refactoring towards rewards from partner teams like H2O / PSDN or Filecoin / FIL. We never completed that because DF volumes never got high enough to justify it. It won't be anytime soon, either. Therefore let's take the opportunity to remove "-OCEAN" from the existing rewards csv filename, and not include "-OCEAN" in the new rewards csv filenames too.

Also, the "rewardsperlp" in "rewardsperlp-OCEAN.csv" assumes that all of DF is Volume DF. We should rename that.

Overall, let's use this PR to set clean csv _rewards_ filenames. I recommend: 
- Volume DF: "volume_rewards.csv" (Rename from rewardsperlp-OCEAN.csv)
- Predictoor DF: "predictoor_rewards.csv"
- Challenge DF: "challenge_rewards.csv"

Let's also have clean csv _data_ filenames. I recommend:
- Volume DF: has many files. Leave them as-is.
- Predictoor DF: call it "predictoor_data.csv" (I recommend this in a comment above as well)
- Challenge DF: call it "challenge_data.csv"